src/file-processor.ts

```diff
@@ -0,0 +1,18 @@
+import {readFile} from "./utils/file-utils";
+
+export default class FileProcessor {
```

Comment:
The method currently located in FileProcessor is a small utility function, why does it need it's own class? Maybe this should be integrated with the renderer somehow?

Reply:
Or you could make the method static